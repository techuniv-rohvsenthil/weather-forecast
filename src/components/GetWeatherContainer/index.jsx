import React from 'react';
import * as styles from './index.module.css';
import Button from '../Button';


const GetWeatherContainer = () => (
  <div className={styles.GetWeatherContainer}>
    <Button testID="test-btn" text="Get Weather" />
  </div>
);

export default GetWeatherContainer;
