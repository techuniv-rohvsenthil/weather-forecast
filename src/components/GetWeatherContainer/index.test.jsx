import React from 'react';
import { render } from '@testing-library/react';

import GetWeatherContainer from './index';


describe('the GetWeatherContainer component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<GetWeatherContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
