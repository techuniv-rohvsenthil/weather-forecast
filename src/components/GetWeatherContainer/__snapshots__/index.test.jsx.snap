// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the GetWeatherContainer component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <div>
      <button
        data-testid="test-btn"
        type="button"
      >
        Get Weather
      </button>
    </div>
  </div>
</DocumentFragment>
`;
